### PR TITLE
[RFC][UN-6690] Fabricator style guide

### PR DESCRIPTION
**Pull request by asantos00**
https://uniplaces.atlassian.net/browse/UN-6690

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
-_site
-.sass-cache
-.jekyll-metadata
+node_modules/
+dist
+src/assets/uniq
```
Comment (mmelo) [inline .gitignore:3]: delete .DS_Store file and add it to .gitignore list

File: bower.json [hunk 2]
```diff
@@ -6,17 +7,19 @@
   ],
   "description": "Uniplaces CSS Framework/Toolkit Documentation",
   "main": "",
-  "moduleType": [
-    "es6",
-    "globals"
+  "authors": [
+    "Luke Askew"
```
Comment (mmelo) [inline bower.json:11]: wrong! 👀

File: package.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  },
+  "keywords": [
+    "toolkit",
+    "styleguide",
+    "framework",
+    "scaffold",
+    "patterns",
+    "library"
+  ],
+  "author": "Luke Askew",
```
Comment (mmelo) [inline package.json]: nop 👀

File: package.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "fabricator",
```
Comment (mmelo) [inline package.json]: `"name": "uniq-wiki"`

File: src/data/uniq.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+    "name" : "uniQ",
+    "moto" : "",
+    "description" : "",
+    "available-at-github" : "The code is available at "
```
Comment (mmelo) [inline src/data/uniq.json]: add missing link here please: https://github.com/uniplaces/uniplaces-uniq

File: src/materials/components/inputs/001-basic.html [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<input type="text" value="Braga" class="input">
+
+
```
Comment (mmelo) [inline src/materials/components/inputs/001-basic.html]: remove extra line

File: src/materials/components/inputs/002-basic-error.html [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<input type="text" value="Braga" class="input input--error">
+
+
```
Comment (mmelo) [inline src/materials/components/inputs/002-basic-error.html]: remove extra line

File: src/materials/components/inputs/003-basic-warning.html [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<input type="text" value="Braga" class="input input--warning">
+
+
```
Comment (mmelo) [inline src/materials/components/inputs/003-basic-warning.html]: remove extra line

File: src/materials/components/inputs/004-basic-success.html [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<input type="text" value="Braga" class="input input--success">
+
+
```
Comment (mmelo) [inline src/materials/components/inputs/004-basic-success.html]: remove extra line

File: src/materials/components/inputs/02-bordered.html [hunk 10]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (mmelo) [inline src/materials/components/inputs/02-bordered.html]: remove extra line at top

File: src/materials/components/inputs/04-price.html [hunk 11]
```diff
@@ -0,0 +1,5 @@
+<div class="input-price">
+  <input type="text" maxlength="4" placeholder="00" value="">
+  <div class="input-price__value"><span class="input-price__currency">€</span> / <span class="input-price__interval">month</span></div>
+</div>
+
```
Comment (mmelo) [inline src/materials/components/inputs/04-price.html]: remove extra line

File: src/materials/components/inputs/05-right-icon-error.html [hunk 12]
```diff
@@ -0,0 +1,10 @@
+<div class="input-container">
+    {{> inputs.basic}}
+    <div class="input-right-icon input-right-icon--error">
+          <?xml version="1.0" encoding="utf-8"?>
+          <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="9.4px" height="9.6px" viewBox="0 0 9.4 9.6" style="enable-background:new 0 0 9.4 9.6;" xml:space="preserve" fill="#FFFFFF">
+          <path class="st0" d="M6.2,4.8l2.5-2.4c0.4-0.4,0.4-1,0-1.4c-0.4-0.4-1-0.4-1.4,0L4.8,3.3L2.4,0.8C2,0.4,1.4,0.4,1,0.8c-0.4,0.4-0.4,1,0,1.4l2.4,2.5L0.9,7.2c-0.4,0.4-0.4,1,0,1.4C1,8.8,1.3,8.9,1.6,8.9c0.3,0,0.5-0.1,0.7-0.3l2.5-2.4l2.4,2.5C7.4,8.9,7.7,9,8,9c0.3,0,0.5-0.1,0.7-0.3c0.4-0.4,0.4-1,0-1.4L6.2,4.8z"/>
+          </svg>
+    </div>
+</div>
+
```
Comment (mmelo) [inline src/materials/components/inputs/05-right-icon-error.html]: remove extra line

File: src/materials/components/inputs/05-right-icon-success.html [hunk 13]
```diff
@@ -0,0 +1,10 @@
+<div class="input-container">
+    {{> inputs.basic}}
+    <div class="input-right-icon input-right-icon--success">
+         <?xml version="1.0" encoding="utf-8"?>
+          <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="11.2px" height="9.2px" viewBox="0 0 11.2 9.2" style="enable-background:new 0 0 11.2 9.2;" xml:space="preserve" fill="#FFFFFF">
+          <g><path class="st0" d="M3.5,8.7C3.2,8.7,3,8.6,2.8,8.4L0.7,6.3c-0.4-0.4-0.4-1,0-1.4s1-0.4,1.4,0l1.4,1.4l5.6-5.2c0.4-0.4,1-0.4,1.4,0c0.4,0.4,0.4,1,0,1.4L4.2,8.4C3.9,8.6,3.7,8.7,3.5,8.7z"/></g>
+          </svg>
+    </div>
+</div>
+
```
Comment (mmelo) [inline src/materials/components/inputs/05-right-icon-success.html]: remove extra line

File: src/materials/components/inputs/05-right-icon-warning.html [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+    {{> inputs.basic}}
+    <div class="input-right-icon input-right-icon--warning">
+        <?xml version="1.0" encoding="utf-8"?>
+        <svg xmlns="http://www.w3.org/2000/svg"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        width="22px" height="22px" viewBox="0 0 22 22">
+        <path fillRule="evenodd" d="M 11 22C 4.93 22-0 17.07-0 11-0 4.93 4.93 0 11 0 17.07 0 22 4.93 22 11 22 17.07 17.07 22 11 22ZM 11 1.76C 5.91 1.76 1.76 5.91 1.76 11 1.76 16.09 5.91 20.24 11 20.24 16.09 20.24 20.24 16.09 20.24 11 20.24 5.91 16.09 1.76 11 1.76ZM 11 18.04C 10.27 18.04 9.68 17.45 9.68 16.72 9.68 15.99 10.27 15.4 11 15.4 11.73 15.4 12.32 15.99 12.32 16.72 12.32 17.45 11.73 18.04 11 18.04ZM 11 13.64C 10.51 13.64 10.12 13.25 10.12 12.76 10.12 12.76 10.12 4.4 10.12 4.4 10.12 3.91 10.51 3.52 11 3.52 11.49 3.52 11.88 3.91 11.88 4.4 11.88 4.4 11.88 12.76 11.88 12.76 11.88 13.25 11.49 13.64 11 13.64Z"/></svg>
+    </div>
+</div>
+
```
Comment (mmelo) [inline src/materials/components/inputs/05-right-icon-warning.html]: remove extra line

File: src/materials/components/inputs/05-right-icon.html [hunk 15]
```diff
@@ -0,0 +1,10 @@
+<div class="input-container">
+    {{> inputs.basic}}
+    <div class="input-right-icon">
+         <?xml version="1.0" encoding="utf-8"?>
+          <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="11.2px" height="9.2px" viewBox="0 0 11.2 9.2" style="enable-background:new 0 0 11.2 9.2;" xml:space="preserve" fill="#FFFFFF">
+          <g><path class="st0" d="M3.5,8.7C3.2,8.7,3,8.6,2.8,8.4L0.7,6.3c-0.4-0.4-0.4-1,0-1.4s1-0.4,1.4,0l1.4,1.4l5.6-5.2c0.4-0.4,1-0.4,1.4,0c0.4,0.4,0.4,1,0,1.4L4.2,8.4C3.9,8.6,3.7,8.7,3.5,8.7z"/></g>
+          </svg>
+    </div>
+</div>
+
```
Comment (mmelo) [inline src/materials/components/inputs/05-right-icon.html]: remove extra line

File: src/materials/components/pills/pill-error.html [hunk 16]
```diff
@@ -0,0 +1,2 @@
+<div class="pill pill--error">error label</div>
+
```
Comment (mmelo) [inline src/materials/components/pills/pill-error.html]: remove extra line

File: src/materials/components/pills/pill-info.html [hunk 17]
```diff
@@ -0,0 +1,2 @@
+<div class="pill pill--info">info label</div>
+
```
Comment (mmelo) [inline src/materials/components/pills/pill-info.html]: remove extra line

File: src/materials/components/pills/pill-success.html [hunk 18]
```diff
@@ -0,0 +1,2 @@
+<div class="pill pill--success">success label</div>
+
```
Comment (mmelo) [inline src/materials/components/pills/pill-success.html]: remove extra line

File: src/materials/components/pills/pill-warning.html [hunk 19]
```diff
@@ -0,0 +1,2 @@
+<div class="pill pill--warning">warning label</div>
+
```
Comment (mmelo) [inline src/materials/components/pills/pill-warning.html]: remove extra line

File: src/materials/components/radio-button/radio-button.html [hunk 20]
```diff
@@ -0,0 +1,7 @@
+<div class="input-container input-container--bordered">
+  <div class="radio-button">
+    <input type="radio" id="piped">
+    <label for="piped">Yes, piped gas</label>
+  </div>
+</div>
+
```
Comment (mmelo) [inline src/materials/components/radio-button/radio-button.html]: remove extra line

File: src/materials/components/textarea/textarea.html [hunk 21]
```diff
@@ -0,0 +1,3 @@
+<textarea name="" id="" rows="5" class="textarea" placeholder="I.E: Two bedroom apartment with a balcony and great view in trendy Bairro Alto."></textarea>
+<div class="textarea-counter">300 characters</div>
+
```
Comment (mmelo) [inline src/materials/components/textarea/textarea.html]: remove extra line

File: src/materials/components/toggle/toggle.html [hunk 22]
```diff
@@ -0,0 +1,19 @@
...
+      <div class="toggle__button"></div>
+      <?xml version="1.0" encoding="utf-8"?>
+      <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="9.4px" height="9.6px" viewBox="0 0 9.4 9.6" style="enable-background:new 0 0 9.4 9.6;" xml:space="preserve" fill="#FFFFFF">
+      <path class="st0" d="M6.2,4.8l2.5-2.4c0.4-0.4,0.4-1,0-1.4c-0.4-0.4-1-0.4-1.4,0L4.8,3.3L2.4,0.8C2,0.4,1.4,0.4,1,0.8c-0.4,0.4-0.4,1,0,1.4l2.4,2.5L0.9,7.2c-0.4,0.4-0.4,1,0,1.4C1,8.8,1.3,8.9,1.6,8.9c0.3,0,0.5-0.1,0.7-0.3l2.5-2.4l2.4,2.5C7.4,8.9,7.7,9,8,9c0.3,0,0.5-0.1,0.7-0.3c0.4-0.4,0.4-1,0-1.4L6.2,4.8z"/>
+      </svg>
+    </div>
+  </div>
+  <label>Full-time workers</label>
+</div>
+
```
Comment (mmelo) [inline src/materials/components/toggle/toggle.html]: remove extra line

File: src/views/index.html [hunk 23]
```diff
@@ -0,0 +1,15 @@
...
+description: A mostly reasonable approach to the CSS and Sass internal toolkit we use at Uniplaces
+github: https://github.com/uniplaces/uniplaces-uniq
+---
+
+<div class="uniplaces-logo"></div>
+<h1>{{name}} - {{moto}}</h1>
+
+<p>{{description}}</p>
+<p>{{github}}</p>
+
```
Comment (mmelo) [inline src/views/index.html]: remove extra line

File: gulpfile.js [hunk 24]
```diff
@@ -0,0 +1,181 @@
...
+  .pipe(prefix('last 1 version'))
+  .pipe(gulpif(!config.dev, csso()))
+  .pipe(gulpif(config.dev, sourcemaps.write()))
+  .pipe(gulp.dest(config.styles.toolkit.dest))
+  .pipe(gulpif(config.dev, reload({ stream: true })));
+});
+
+gulp.task('styles', ['styles:fabricator', 'styles:toolkit']);
+
+
```
Comment (mmelo) [inline gulpfile.js]: remove extra line

File: gulpfile.js [hunk 25]
```diff
@@ -0,0 +1,181 @@
...
+      dest: 'dist/assets/toolkit/images',
+      watch: 'src/assets/toolkit/images/**/*',
+    },
+  },
+  templates: {
+    watch: 'src/**/*.{html,md,json,yml}',
+  },
+  dest: 'dist',
+};
+
```
Comment (mmelo) [inline gulpfile.js:60]: remove extra line

File: gulpfile.js [hunk 26]
```diff
@@ -0,0 +1,181 @@
...
+
+// clean
+gulp.task('clean', del.bind(null, [config.dest]));
+
+
```
Comment (mmelo) [inline gulpfile.js]: remove extra line

File: gulpfile.js [hunk 27]
```diff
@@ -0,0 +1,181 @@
...
+    if (result.errors.length) {
+      result.errors.forEach((error) => {
+        gutil.log(gutil.colors.red(error));
+      });
+    }
+    done();
+  });
+});
+
+
```
Comment (mmelo) [inline gulpfile.js]: remove extra line

File: gulpfile.js [hunk 28]
```diff
@@ -0,0 +1,181 @@
...
+    .pipe(imagemin())
+    .pipe(gulp.dest(config.images.toolkit.dest));
+});
+
+gulp.task('favicon', () => {
+  return gulp.src('src/favicon.ico')
+  .pipe(gulp.dest(config.dest));
+});
+
+
```
Comment (mmelo) [inline gulpfile.js]: remove extra line

File: gulpfile.js [hunk 29]
```diff
@@ -0,0 +1,181 @@
...
+// assembler
+gulp.task('assembler', (done) => {
+  assembler({
+    logErrors: config.dev,
+    dest: config.dest,
+  });
+  done();
+});
+
+
```
Comment (mmelo) [inline gulpfile.js]: remove extra line

File: gulpfile.js [hunk 30]
```diff
@@ -0,0 +1,181 @@
...
+  gulp.task('scripts:watch', ['scripts'], browserSync.reload);
+  gulp.watch([config.scripts.fabricator.watch, config.scripts.toolkit.watch], ['scripts:watch']);
+
+  gulp.task('images:watch', ['images'], browserSync.reload);
+  gulp.watch(config.images.toolkit.watch, ['images:watch']);
+
+});
+
+
```
Comment (mmelo) [inline gulpfile.js]: remove extra line

File: gulpfile.js [hunk 31]
```diff
@@ -0,0 +1,181 @@
...
+    'assembler',
+  ];
+
+  // run build
+  runSequence(tasks, () => {
+    if (config.dev) {
+      gulp.start('serve');
+    }
+  });
+
```
Comment (mmelo) [inline gulpfile.js:173]: remove extra line

File: src/assets/toolkit/styles/alert.scss [hunk 32]
```diff
@@ -0,0 +1,13 @@
...
+    margin-top: 16px;
+}
+
+.pill {
+    @include pill();
+    display: inline-block;
+    margin-right: 16px;
+    margin-bottom: 46px;
+}
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/alert.scss]: remove extra line

File: src/assets/toolkit/styles/alert.scss [hunk 33]
```diff
@@ -0,0 +1,13 @@
...
```
Comment (mmelo) [inline src/assets/toolkit/styles/alert.scss]: remove extra line

File: src/assets/toolkit/styles/animation.scss [hunk 34]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/animation.scss]: remove extra line

File: src/assets/toolkit/styles/animation.scss [hunk 35]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    &--scale-in {
+        transform: scale3d(0.5, 0.5, 0.5);
+        &:hover {
+            @include scale-in();
+        }
+    }
+}
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/animation.scss]: remove extra line

File: src/assets/toolkit/styles/buttons.scss [hunk 36]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/buttons.scss]: remove extra line

File: src/assets/toolkit/styles/colors.scss [hunk 37]
```diff
@@ -0,0 +1,27 @@
...
+    word-break: break-all;
+    @each $arrayName, $colorArray in $palette {
+        @each $colorName, $color in $colorArray {
+            &--#{$arrayName}-#{$colorName} {
+                background: $color;
+            }
+        }
+    }
+}
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/colors.scss]: remove extra line

File: src/assets/toolkit/styles/dropdown.scss [hunk 38]
```diff
@@ -0,0 +1,21 @@
...
+
+    &__wrapper {
+        @include breakpoint(small) {
+            display: none;
+        }
+        width: 100%;
+    }
+
+}
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/dropdown.scss]: remove extra line

File: src/assets/toolkit/styles/input.scss [hunk 39]
```diff
@@ -0,0 +1,49 @@
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/input.scss]: remove extra line

File: src/assets/toolkit/styles/header.scss [hunk 40]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/header.scss]: remove extra line

File: src/assets/toolkit/styles/input.scss [hunk 41]
```diff
@@ -0,0 +1,49 @@
...
+.textarea {
+    @include textarea();
+    width: 100%;
+    
+    &-counter {
+        margin-bottom: 46px;
+    }
+}
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/input.scss]: remove extra lines

File: src/assets/toolkit/styles/messages.scss [hunk 42]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/messages.scss]: remove extra line

File: src/assets/toolkit/styles/progress-bar.scss [hunk 43]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.progress-bar {
+    @include progress-bar();
+    &__bar {
+        @include animateWidth();
+    }
+}
+
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/progress-bar.scss]: remove extra lines

File: src/assets/toolkit/styles/toolkit.scss [hunk 44]
```diff
@@ -0,0 +1,65 @@
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/toolkit.scss]: remove extra line

File: src/assets/toolkit/styles/toolkit.scss [hunk 45]
```diff
@@ -0,0 +1,65 @@
...
+        @include column(6);
+    }
+}
+
+
+.image-gallery {
+    @include image-gallery();
+    margin-bottom: 46px;
+}
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/toolkit.scss]: remove extra line

File: src/assets/toolkit/styles/typography.scss [hunk 46]
```diff
@@ -0,0 +1,51 @@
...
+    &--button-text {
+        @include button-text();
+        text-align: left;
+    }
+    &--anchor {
+        @include anchor();
+    }
+}
+
+
```
Comment (mmelo) [inline src/assets/toolkit/styles/typography.scss]: remove extra line

File: src/data/toolkit.yml [hunk 47]
```diff
@@ -0,0 +1,10 @@
+colors:
+  primary:
+    Dark: "rgb(30, 30, 30)"
+    Light: "rgb(242, 242, 242)"
+  secondary:
+    Red: "rgb(201, 55, 6)"
+    Yellow: "rgb(252, 207, 80)"
+    Blue: "rgb(39, 102, 143)"
+
+
```
Comment (mmelo) [inline src/data/toolkit.yml]: remove extra lines

File: src/materials/components/animations/slide-in.html [hunk 48]
```diff
@@ -0,0 +1,9 @@
+---
+notes: |
+    Hover on the element to see the animation
+---
+
+<div class="animation animation--slide-in">
+    {{> alerts.alert-success }}
+</div>
+
```
Comment (mmelo) [inline src/materials/components/animations/slide-in.html:9]: remove extra line

File: src/materials/components/buttons/011-button-secondary.html [hunk 49]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (mmelo) [inline src/materials/components/buttons/011-button-secondary.html]: remove extra line

File: src/materials/components/buttons/012-button-alternative.html [hunk 50]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (mmelo) [inline src/materials/components/buttons/012-button-alternative.html:1]: remove extra line

File: src/materials/components/buttons/013-button-disabled.html [hunk 51]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (mmelo) [inline src/materials/components/buttons/013-button-disabled.html]: remove extra line

File: src/materials/components/buttons/014-button-upercase.html [hunk 52]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (mmelo) [inline src/materials/components/buttons/014-button-upercase.html]: remove extra line

File: src/materials/components/checkbox/checkbox.html [hunk 53]
```diff
@@ -0,0 +1,20 @@
...
+     <?xml version="1.0" encoding="utf-8"?>
+      <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+               width="15px" height="11.9px" viewBox="0 0 15 11.9" style="enable-background:new 0 0 15 11.9;" xml:space="preserve">
+      <polygon class="st0" points="12.9,0.2 5.5,8 2.4,5 0.7,6.7 5.5,11.5 14.7,1.9 " fill="#FFFFFF"/>
+      </svg> 
+    </div>
+    <label for="">Elevator</label>
+</div>
+
+
```
Comment (mmelo) [inline src/materials/components/checkbox/checkbox.html:12]: remove extra line

File: src/materials/components/checkbox/checkbox.html [hunk 54]
```diff
@@ -0,0 +1,20 @@
...
+<script async>
+(function () {
+  $('.checkbox').on('click', function () {
+    $(this).toggleClass('checkbox--checked');
+  });
+})();
+</script>
+
```
Comment (mmelo) [inline src/materials/components/checkbox/checkbox.html]: remove extra line

File: src/materials/components/footer/footer.html [hunk 55]
```diff
@@ -0,0 +1,7 @@
+<footer id="footer" class="footer footer--overhangs">
+    <div class="footer__wrapper">
+      <button class="button button--align-left">Back</button>
+      <button class="button button--primary button--cool">Next</button>
+    </div>
+</footer>
+
```
Comment (mmelo) [inline src/materials/components/footer/footer.html]: remove extra line

File: src/materials/components/header/header.html [hunk 56]
```diff
@@ -0,0 +1,14 @@
...
+    <div class="header__wrapper">
+      <div class="header__wrapper__logo"></div>
+      <div class="header__wrapper__breadcrumb">
+        <div class="header__wrapper__breadcrumb__step">First step:</div>
+        <div class="header__wrapper__breadcrumb__description">Describe your property</div>
+      </div>
+      <button class="button button--align-right">Exit</button>
+    </div>
+  </header>
+
```
Comment (mmelo) [inline src/materials/components/header/header.html]: remove extra line

**Review by mmelo — CHANGES_REQUESTED** [review]
@asantos00 reviewed

File: bower.json (hunk 2 above)
Comment (asantos00) [reply bower.json:11]: I've asked for you yo review this PR right before I fixed it, sorry

**Review by mmelo — APPROVED** [review]

**Pull request merged**